packages/terra-cell-grid/src/terra-dev-site/doc/example/CellGridExample.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import CellGrid, {
+  Cell,
+} from 'terra-cell-grid';
```

Comment:
Just curious, is your IDE linter auto formatting these or is your preference to new line all brackets?

```jsx
import CellGrid, {
  Cell,
} from 'terra-cell-grid';
```

vs

```jsx
import CellGrid, { Cell } from 'terra-cell-grid';
```

Reply:
Just carried over the pattern from other examples. A copy paste I decided to leave as is.